src/transformers/modeling_tf_utils.py

```diff
@@ -616,7 +613,6 @@ def from_pretrained(cls, pretrained_model_name_or_path, *model_args, **kwargs):
         proxies = kwargs.pop("proxies", None)
         output_loading_info = kwargs.pop("output_loading_info", False)
         local_files_only = kwargs.pop("local_files_only", False)
-        use_cdn = kwargs.pop("use_cdn", True)
```

Comment:
No need to pop revision here?

Reply:
Yes, will also add to TF/Flax/etc (cc @LysandreJik) before merging.